src/main/kotlin/main.kt

```diff
@@ -16,6 +18,9 @@ fun main() {
     val jsonGenerator = JsonGenerator()
     val serverError = InternalServerError()
     val fileIo = FileIo()
+    val filepath = "src/main/kotlin/resources/task-list.txt"
```

Comment:
It would be nice if this value (and server port) could be configured from outside the code. I think we tried to get environment variables working and couldn't figure it out. So, I think it's okay to leave this as is for now.